src/crypto/crypto_context.cc

```diff
@@ -878,7 +878,7 @@ void StartLoadingCertificatesOffThread(
   // loading.
   {
     Mutex::ScopedLock cli_lock(node::per_process::cli_options_mutex);
-    if (!per_process::cli_options->ssl_openssl_cert_store) {
+    if (per_process::cli_options->ssl_openssl_cert_store) {
```

Comment:
Would you mind adding a comment to this line?

Reply:
The comment explaining it is above the block (L876-878)